### DISCUSSION TITLE
Carbs are not synchronized to xDrip sync follower (AAPS 3.0)

### DISCUSSION THREAD

**Discussion by vmichalec** [NightscoutFoundation/xDrip#1935] [General]
Hi, I'm using AAPS 3.0 (still beta) with xDrip that accepts treatments. On master phone it works like it should. I have also xDrip sync follower on another phone but here is the problem. Carbs are not shown here.

AAPS
![AAPS](https://user-images.githubusercontent.com/24793786/149627781-a7b3f323-07f0-48dd-953d-c16414fb409b.png)

xDrip sync master - here with insulin and carbs shown
![xDrip-master](https://user-images.githubusercontent.com/24793786/149627789-608891e0-2f12-4901-a116-02e42768077d.png)

xDrip sync follower - here are only insulin entries but no carbs
![xDrip-sync-follower](https://user-images.githubusercontent.com/24793786/149627799-3cbd8ec3-6b39-4675-adb9-4a4d90e097d9.jpg)

Nightscout treatments - entries as seen on nightscout
![nightscout-treatments](https://user-images.githubusercontent.com/24793786/149627811-f9e07fd8-aee1-495a-aaab-11d1effa490f.jpg)

I have checked it on 2021.12.24 version of xDrip. (It was working with previous AAPS versions.)

( This is a copy of https://github.com/Navid200/xDrip/discussions/178 )

**Comment by psonnera** [comment]
If you manually add a dummy carb entry (like 1) on the master xDrip+ does it show on the xDrip+ follower?

**Reply by vmichalec** [reply]
I just created carb entry in AAPS on phone with xDrip master and this entry appeared in xDrip master and also on follower phone.
